### PR TITLE
PROCSE-2.0-goolf-1.4.10.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/5ee17940c25525c39908 for a full test report.

File: easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+name = 'PROCSE'
+version = '2.0'
+
+homepage = 'http://www.biozentrum.unibas.ch/~nimwegen/cgi-bin/procse.cgi'
+description = """PROCSE (PRObabilistic Clustering of Sequences) is a Bayesian algorithm 
+ for clustering sequences over a finite alphabet (designed specifically for DNA sequences) 
+ into groups using a mixture model that assumes all sequences in a group derive from a common 
+ position-specific weight matrix (which is automatically integrated out of the likelihood). 
+ The program uses Markov chain Monte-Carlo sampling to sample the space o all partitions of
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb]: o -> of
forDNA -> for DNA (below)

File: easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+source_urls = ['http://www.biozentrum.unibas.ch/~nimwegen/procse/']
+sources = ['%(namelower)s%(version)s.tar.gz']
+
+parallel = 1
+
+files_to_copy = [
+                (["procse"], "bin"),
+                "minimal_params", "README", "params_plus_explanation"
+]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb]: 4-space indentation.

this PR has only style remarks and is ready for test/merge

Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PROCSE/PROCSE-2.0-goolf-1.4.10.eb]: minor changes needed here , put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**